kuma/javascript/src/page.jsx

```diff
@@ -245,7 +245,9 @@ function Article() {
             <div className="text-content" css={styles.article}>
                 <article
                     id="wikiArticle"
-                    dangerouslySetInnerHTML={{ __html: documentData.bodyHTML }}
+                    dangerouslySetInnerHTML={{
+                        __html: documentData.bodyHTML
+                    }}
```

Comment:
I think you might need to re-run `npm run format` to put this line back the way it was. Since the text has not changed, I think the formatting should not have either.